db.py

```diff
@@ -0,0 +1 @@
+db = {}
```

Comment:
When working with databases, there is really famous and beautiful pattern called `Repository Pattern`.
It abstracts the underlying databases structure from other part of the code. I would suggest you to use that.

For example, you will create an interface which has `persist_to_db(data)` method. Then your APIs accept that interface and call that method when they want to write something in db. Then you can easily have different DBs without the need of changing your main business.

Please check more here:

- https://python.plainenglish.io/design-patterns-in-python-repository-pattern-1c2e5070a01c
- https://samjoness92332.medium.com/repository-pattern-is-insane-if-you-know-how-to-use-it-properly-python-8fc0dff90075